torch/distributed/elastic/agent/server/local_elastic_agent.py

```diff
@@ -254,6 +263,17 @@ def _start_workers(self, worker_group: WorkerGroup) -> Dict[int, Any]:
             if "OMP_NUM_THREADS" in os.environ:
                 worker_env["OMP_NUM_THREADS"] = os.environ["OMP_NUM_THREADS"]
 
+            if "TORCHELASTIC_LOG_LINE_HEADER" in os.environ:
```

Comment:
for the sake of keeping the elastic agent programmatic, this be taken as a ctor argument. The `TORCHELASTIC_LOG_LINE_HEADER` should be parsed in the `torchrun` CLI (e.g. `torch/distributed/run.py`)

Reply:
Good suggestion, done.